test/Interop/Cxx/operators/member-inline-silgen.swift

```diff
@@ -32,3 +32,16 @@ public func call(_ wrapper: inout AddressOnlyIntWrapper) -> Int32 { wrapper() }
 // CHECK: end_access [[SELFACCESS]] : $*AddressOnlyIntWrapper
 
 // CHECK: sil [clang AddressOnlyIntWrapper.callAsFunction] [[NAME]] : $@convention(c) (@inout AddressOnlyIntWrapper) -> Int32
+
+public func index(_ wrapper: inout ReadOnlyIntArray, _ arg: Int32) -> Int32 { wrapper[arg] }
+
```

Comment:
Always make sure you check the function name and signature before the function body.

Reply:
Done, if I understood you correctly